app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_display_set_update.html

```diff
@@ -1,10 +1,22 @@
 {% load crispy_forms_tags %}
 
 <div class="card-body py-1" id="ds-content-{{ object.id }}">
+   {% if messages %}
+    {% for message in messages %}
+        <div class="alert alert-{{ message.level_tag }}">
+            {{ message }}
+            <button type="button" class="close"
+                    data-dismiss="alert" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+            </button>
+        </div>
+    {% endfor %}
+  {% endif %}
   {% if object.is_editable %}
-    <form action="{% url 'reader-studies:display-set-update' pk=object.pk %}" method="post" class="ds-form" data-hx-get="{% url 'reader-studies:display-set-detail' pk=object.pk %}" data-hx-target="#ds-content-{{ object.id }}" data-hx-swap="outerHTML">
+    <form action="{% url 'reader-studies:display-set-update' pk=object.pk %}" method="post" class="ds-form" data-hx-get="{% url 'reader-studies:display-set-detail' pk=object.pk %}" data-hx-target="#ds-content-{{ object.id }}" data-hx-swap="outerHTML" id="form-{{ object.pk }}">
```

Comment:
Why do you need the `data-hx-get` attribute here? This does a post request to the ds update view, which returns the ds detail view on success. So then this attribute should be redundant here, right?

Reply:
You're right, we only need the `data-hx-target` attribute, that is used here: https://github.com/comic/grand-challenge.org/blob/main/app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_images_list.html#L96. This form gets processed using js :slightly_smiling_face: